src/__test__/components/data-exploration/embedding/CrossHair.test.jsx

```diff
@@ -11,7 +11,7 @@ describe('CrossHair', () => {
   test('renders correctly', () => {
     const store = mockStore({
       cellInfo: {
-        cellName: 1904,
+        cellId: 1904,
```

Comment:
While you're making this change, can you also fix the relative import paths to use absolute import paths in this file?